app/controllers/api/v1/testimonials_controller.rb

```diff
@@ -3,6 +3,17 @@
 module Api
   module V1
     class TestimonialsController < ApplicationController
+      before_action :set_testimonial, only: %i[destroy]
+      def destroy
+        @testimonial.discard
+        head :no_content
+      end
+
+      private
+
+      def set_testimonial
+        @testimonial = Testimonial.find(params[:id])
```

Comment:
Would be better to use the method `kept` to check if the Testimonial was previously discarded:
```suggestion
        @testimonial = Testimonial.kept.find(params[:id])
```